src/content/docs/journey-migration/migrating-dashboards.mdx

```diff
@@ -17,9 +17,9 @@ import journeyAzureDash from 'images/infrastructure_screenshot-crop_journey-guid
 
 import journeyAlertsAndDash from 'images/dashboards_screenshot-crop_journey-dash-and-alerts.png'
 
-Setting up monitors for your data is the foundation of observability, but it's only half of the equation. You need to be able to visualize your data, group streams together, and be notified when your system runs into issues that negatively affect your users. You need to be able to do all this from a centralized location, and be able to adapt to the changing data needs of your organization by curating new visualizations on-the-fly. 
+Setting up monitors for your data is the foundation of observability, but it's only half of the equation. You need to be able to visualize your data, group data points together, and be notified when your system runs into issues. You need to be able to do all this from a centralized location, and be able to adapt to the changing data needs of your organization by creating new visualizations on-the-fly. 
 
-Where New Relic entities can give you scores of data to observe, our dashboards and alerts come in to help you make the most of it. You can use our pre-configured options to setup solutions out of the box, or use [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/), New Relic's Query Language, to create custom dashboard components and alert policies that are as fine-tuned to your requirements as you need want them to be. And like all our other features, you can try out dashboards and alerts using your own data [for free](https://newrelic.com/signup).
+Where New Relic entities can give you scores of data to observe, our dashboards and alerts come in to help you make the most of it. You can use our pre-configured options to setup solutions, or use [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/), New Relic's Query Language, to create custom dashboard components and alert policies that are as fine-tuned to your requirements. And like all our other features, you can try out dashboards and alerts using your own data [for free](https://newrelic.com/signup).
```

Comment:
"scores of data" feels odd. How about "scores of data sources" or "scores of applications and their data" or even some other way of indicating a large number?

Reply:
Pretty sure it should be "set up"

Reply:
"...fine-tuned to your requirements as you need" I think would be slightly better

Reply:
Fixed em all. Thank you!